scripts/upload-source-maps.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+import { execSync } from 'child_process'
+import { readdirSync, unlinkSync } from 'fs'
+import { join } from 'path'
+
+const main = () => {
+	if (process.env.VERCEL_ENV === 'development') {
+		return
+	}
+
+	const LATEST_SHA = process.env.VERCEL_GIT_COMMIT_SHA
+	// const PATH_PREFIX = 'https://developer.hashicorp.com/_next/static/'
+	// const SERVICE = 'developer.hashicorp.com'
+	const PATH_PREFIX =
+		'https://dev-portal-git-rn-featbuild-source-maps-during-deploy-hashicorp.vercel.app/_next/static/'
+	const SERVICE = 'non-prod.developer.hashicorp.com'
```

Comment:
I think this'll need to be variable based on deployment environment. Something like:

```suggestion
	const SERVICE = process.env.VERCEL_ENV === 'production' ? 'developer.hashicorp.com' : 'non-prod.developer.hashicorp.com'
```

Reply:
Good call!